scripts/.eslintrc.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+# Necessary to support async/await... grumble grumble...
+parserOptions:
+  ecmaVersion: 8
+
+rules:
+  space-before-function-paren:
```

Comment:
Can you modify eslintrc in root to match this?

Reply:
Sure thing! It's kinda clumsy though in the current shape, do you mind if I change overall into `space-before-function-paren: error` (i.e. `always`)? It's seems way more common to me and ESLint's default.
Plus I can do that automatically with `--fix` to not break anything.

Will do in another PR if you're okay with this.

Reply:
Oh, that's gonna affect so much that I wouldn't recommend doing it.

Reply:
Yeah, about 400 `function()`... I really hate those, but whatever.

Anyway, moved this to the root config!